src/software/firmware/srcs/respirator.cpp

```diff
@@ -148,7 +148,7 @@ void setup(void) {
     lastpControllerComputeDate = millis();
 
     // Catch potential Watchdog reset
-    if (IWatchdog.isReset(true)) {
+    if (IWatchdog.isReset(true) == true) {
```

Comment:
CppCheck doesn't have access to Stm32duino headers, this is a false positive.
Keep 
```suggestion
    if (IWatchdog.isReset(true)) {
```